### PR TITLE
Added Social Media Links 

### PR DESCRIPTION
**Pull request by hkaur008**
Things added so far:-
- Add a column for social media where the link of the same can be added.
- Add demo data for social media accordingly.
- Added a small custom CSS.
- Added the section on the home page to link to the social media section.

Reasons to add social media links;-
- To help viewers track the progress of the user.

## Please, go through these steps before you submit a PR.

- [X] You've explained what the Pull Request is adding.
- [X] You've explained why this is important.

Fixed issue no #2

### CODE REVIEW

File: _sass/all.scss [hunk 1]
```diff
@@ -76,6 +76,11 @@ a {
         font-size: small;
         color: #404040;
     }
+    
+    .username {
+        font-size: medium;
+        color: #404040;
```
Comment (rosekamallove) [inline _sass/all.scss:82]: Changin hard-coded color to variables would be better

**Review by rosekamallove — CHANGES_REQUESTED** [review]
Please Change the hardcoded color values to one of the color variables in _sass/theme.scss

File: _sass/all.scss (hunk 1 above)
Comment (hkaur008) [reply _sass/all.scss:82]: hey, the changes made by you will be inherited in our pull requests,  once your pull request gets merged! 
no need to make changes if made in your pull request

**Comment by awilkescom** [comment]
@hkaur008 very nice - but before I merge, it seems like @rosekamallove has a valid suggestion, could we add those?

**Comment by hkaur008** [comment]
> @hkaur008 very nice - but before I merge, it seems like @rosekamallove has a valid suggestion, could we add those?
Actually @shivamsouravjha  worked over this part let me ask him

**Pull request merged**